src/test/java/org/opensearch/neuralsearch/search/query/HybridAggregationProcessorTests.java

```diff
@@ -84,37 +44,23 @@ public void testAggregationProcessorDelegate_whenPreAndPostAreCalled_thenSuccess
     }
 
     @SneakyThrows
-    public void testCollectorManager_whenHybridQueryAndNotConcurrentSearch_thenSuccessful() {
```

Comment:
Now that we have rely on HybridQueryCollectorContextSpec to handle concurrent searches, how can we test concurrent searches in hybrid search? Through ITs probably

Reply:
Actually we rely on ConcurrentQueryPhaseSearcher